net/lksctp-tools/Makefile

```diff
@@ -73,9 +71,9 @@ define Package/libsctp/install
 		$(1)/usr/lib/
 endef
 
-define Package/sctp/install
-	:
-endef
+#define Package/sctp/install
+#	:
+#endef
```

Comment:
If you are going to remove something - remove it. Don't just comment stuff out. Doing so makes the sources too messy. If you want to see what was removed use diff. (git log -p)

Reply:
Testing to see what makes the buildbot tick.

Reply:
Test it in your own environment - Use SDK to build and you'll get the same results. Use the .circleci/config.yml script as a guide if you want a procedure. The packages buildbots are a completely different environment which builds ALL modules and no images.